sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -64,7 +72,7 @@ typedef struct
  * @param endpoint A url to a blob storage account.
  * @param credential The object used for authentication.
  *         #AZ_CREDENTIAL_ANONYMOUS should be used for SAS.
- * @param options  A reference to an #az_storage_blobs_blob_client_options structure which defines
+ * @param options A reference to an #az_storage_blobs_blob_client_options structure which defines
```

Comment:
What happens if it is null? We should follow the same pattern as other `options` structs where null == default.
https://github.com/Azure/azure-sdk-for-c/blob/4334e327a73723e32176033d3fe4611c38872f4d/sdk/inc/azure/core/az_json.h#L248-L250